WorkspaceServer.Tests/Kernel/CSharpKernelTests.cs

```diff
@@ -192,9 +192,29 @@ public async Task it_produces_values_when_executing_Console_output()
             KernelEvents.OfType<ValueProduced>()
                 .Should()
                 .BeEquivalentTo(
-                    new ValueProduced("value one", kernelCommand, new[] { new FormattedValue(null, "value one"), }),
-                    new ValueProduced("value two", kernelCommand, new[] { new FormattedValue(null, "value two"), }),
-                    new ValueProduced("value three", kernelCommand, new[] { new FormattedValue(null, "value three"), }));
+                    new ValueProduced("value one", kernelCommand, false, new[] { new FormattedValue(null, "value one"), }),
+                    new ValueProduced("value two", kernelCommand, false, new[] { new FormattedValue(null, "value two"), }),
+                    new ValueProduced("value three", kernelCommand, false, new[] { new FormattedValue(null, "value three"), }));
```

Comment:
myme types are done later on when formatting on raw value is done

Reply:
I think `FormattedValue` without a mime type is confusing and probably bug prone. I think this test shows that because of the redundancy -- the formatted values ARE the raw values.

Reply:
null value is generated by the  [Formatter.MimeTypeFor](https://github.com/dotnet/try/blob/0fd2e197e62d54090c591584b3a392bf42fcc7d8/Microsoft.DotNet.Interactive.Rendering/Formatter.cs#L361) method

Reply:
That's the right behavior. `FormattedValue.MimeType` is intentionally read-only because the expectation was that the mime type must be known at the time a `FormattedValue` is created. It was an oversight that the constructor didn't guard against this.